cmd/connect-all.go

```diff
@@ -54,6 +55,9 @@ func newConnectAllCmd() *cobra.Command {
 	cmd.Flags().IntP("max-queues", "m", 0, "max-queues")
 	viper.BindPFlag("connect-all.max-queues", cmd.Flags().Lookup("max-queues"))
 
+	cmd.Flags().IntP("ctrl-loss-tmo", "", -1, "controller loss timeout period (in seconds). Timeout is disabled by default (-1)")
```

Comment:
@yogev-lb what is the usecase where it shouldn't set to -1 ?

Reply:
@maor-lb we noticed, as part of the DMS service development that when the volume we are running/writing from/to
is reaching `Unavailable` state we are blocked on I/O and will not return.
This behavior for our needs can lead to hours of hanging workflow, that we would like to abort due to transient error at some point and let the user re-trigger it, once the volume becomes `Available`

Reply:
@yogev-lb so you are saying that read/write would get stuck indefinitely rather than timing out if volume is unavailable/Read-only?
does this happen for the read/write/compare?
shouldn't the timeout be at the application level?
how/what is our retry for this flow?  issue might be intermittent.
is your plan to have a timeout of say 5m and then have non retrievable error?
in this scenario what assures us that a new nvme connect would be issued?
DS doesnt send notification to DC due to changes in protection state does it?

Reply:
@ronen-lb as discussed y, the io will block on randomally read/write operation.
there is no retry here, we wait for `1m` if no connection to controller we will the job, and roll-back
all resources, the user will need to retry this opeation later.

in case all is fixed we will still be connected to the server and a new volume will appear.
the issue will not affect the connectivity to other On-Line DL servers or to other clusters.

i think the DS does send to the DC a notification, but this is after the DC did it's job.
we are talking post DS/DC comm. we are in the io phase.

Reply:
That means that if IO is stuck for 1m you going to fail the all operation ?
I'm sensing potential bug report , I hope time is configurable